cmd/snap-bootstrap/partition/sfdisk.go

```diff
@@ -80,7 +80,7 @@ type DeviceStructure struct {
 // NewDeviceLayout obtains the partitioning and filesystem information from the
 // block device.
 func DeviceLayoutFromDisk(device string) (*DeviceLayout, error) {
-	output, err := exec.Command("sfdisk", "--json", "-d", device).CombinedOutput()
+	output, err := exec.Command("sfdisk", "--json", "-d", device).Output()
```

Comment:
We won't have stderr in the `OutputErr()` below but it should be fine because it will still display `err`.

Reply:
Yeah, I checked the docs, it says that err is populated with stderr if Output is used and there is a non-zero exit code.